src/datasets/label_utils.py

```diff
@@ -160,6 +169,7 @@ class MapLabelsFunction(Enum):
     MULTIPLEX_CELL_LINES = (get_cell_lines_from_multiplex_labels,)
     MULTIPLEX_CELL_LINES_CONDITIONS = (get_cell_lines_conditions_from_multiplex_labels,)
     REMOVE_MARKER = (remove_markers,)
+    OPENCELL = (opencell_map,)
```

Comment:
Where do you use it?
And please adjust it to something like OPENCELL_MAPPING. When used it convert from marker name to localization, right?